src/main/java/com/jcraft/jsch/Channel.java

```diff
@@ -43,8 +44,8 @@ LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
   static final int SSH_OPEN_UNKNOWN_CHANNEL_TYPE=           3;
   static final int SSH_OPEN_RESOURCE_SHORTAGE=              4;
 
-  static int index=0; 
-  private static Vector<Channel> pool=new Vector<>();
+  private static AtomicInteger index = new AtomicInteger(0);
+  private static ConcurrentHashMap<Integer, Channel> pool = new ConcurrentHashMap<>(16, 0.75f, 1);
```

Comment:
1. This should be `ConcurrentMap<Integer, Channel> pool` instead of `ConcurrentHashMap<Integer, Channel> pool`.
2. Can you provide some reasoning for why these particular values were chosen for the ConcurrentHashMap constructor?